clean.sh

```diff
@@ -36,6 +36,7 @@ docker rmi \
     php_grpc_test \
     mcr.microsoft.com/dotnet/core/sdk:3.1 \
     csharp_grpc_test \
+    dotnet_grpc_test \
```

Comment:
Please add the base image (`mcr.microsoft.com/dotnet/sdk:5.0`) before this line 😉